### PR TITLE
Homework4

### PR DESCRIPTION
**Pull request by MalekSafory**
Validations homework
sorry for being late :)

### CODE REVIEW

File: hw4/js/javas.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+// age
+function checkAge(age) {
+    if (age == '')
+        return false;
+    if (age < 120)
+        return true;
+    else return false;
+}
+// name
+function checkName(name) {
```
Comment (EliasSalom) [inline hw4/js/javas.js:59]: very complex function, please review the function and write it again!!

File: hw4/js/javas.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        if (c > 0 && email[i] == '.')
+            dot = i;
+    }
+    if (dot < email.length && dot != 0) {
+        return true;
+    }
+    return false;
+}
+// id
+function checkId(id) {
```
Comment (EliasSalom) [inline hw4/js/javas.js:18]: the function is not working will, please fix the function

**Review by EliasSalom — CHANGES_REQUESTED** [review]
after reviewing the js file I have been put some commitments on the file changed all the issues and pushed the new code

File: hw4/js/javas.js (hunk 1 above)
Comment (MalekSafory) [reply hw4/js/javas.js:42]: i have rewrote it with simpler code

File: hw4/js/javas.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+function checkAge(age) {
+    if (age == '')
+        return false;
+    if (age < 120)
+        return true;
+    else return false;
+}
+// name
+function checkName(name) {
+    let arr = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"];
```
Comment (EliasSalom) [inline hw4/js/javas.js:43]: please check this line, this is what make all the functions complex, please change it or use it in different way

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please check my comments

File: hw4/js/javas.js (hunk 3 above)
Comment (MalekSafory) [reply hw4/js/javas.js]: done , i have changed the function and made it more simple

**Review by EliasSalom — APPROVED** [review]
Good work

**Pull request merged**